docs/slack.rst

```diff
@@ -53,7 +53,16 @@ Chances are we have a large group of folks in your neck of the woods,
 so join up and chat with them.
 They're a great way to get in touch with your local meetup organizer and community.
 
-Social Rules
+Rules
+-----
+
+We have a few hard and fast rules that we ask people to follow on our Slack:
+
+- **No Direct Pitches** - If you’re coming here *only* to make a sales pitch, please don’t. If you just put a link in the chat without any context or conversation, we will delete it.
+- **No Brands** - If you're going to post, post as yourself, not as a brand. We are a community of individuals and we all exist outside of our day jobs and personal brands.
+- **Use inclusive language** - Our community is multicultural, multiracial, multigendered, and generally diverse. When posting in channels, avoid using words such as "guys" to address a mixed group. Possible alternatives: "folks", "everyone", "documentarians".
+
```

Comment:
Context: from a [discussion of a previous issue](https://github.com/writethedocs/www/issues/496), from @Bradamant3 

"TL;DR: promoting your own ideas or thoughts for larger discussion is one thing, promoting your commercial product or service is another (the latter is advertising and if we're going to allow it we should charge for it)."